### PR TITLE
[#157596540] detailed results

### PR DESCRIPTION
**Pull request by andrewjstrominger**

### CODE REVIEW

File: resources/schema/spidacalc/calc/design.schema [hunk 1]
```diff
@@ -32,32 +32,48 @@
...
             "items": {
-                "additionalProperties": false,
-                "description": "A collection of results for a loading or strength case.",
-                "properties": {
-                    "id": {
-                        "description": "an id, usually the loadcase name",
-                        "type": "string"
+                "oneOf": [
+                    {
+                        "id": "results",
```
Comment (andrewjstrominger) [inline resources/schema/spidacalc/calc/design.schema]: Not needed.

File: resources/schema/spidacalc/calc/design.schema [hunk 2]
```diff
@@ -32,32 +32,48 @@
...
-                                "$ref": "../../spidacalc/results/result.schema"
-                              }
-                            ]
-                        }
+                    {
+                        "$ref": "../../spidacalc/results/results.schema"
+                    },
+                    {
+                        "additionalProperties": false,
+                        "description": "Path to a results object, the object should be a /spidacalc/results/results.schema",
```
Comment (andrewjstrominger) [inline resources/schema/spidacalc/calc/design.schema]: the object must be a

File: build.gradle [hunk 3]
```diff
@@ -11,7 +11,7 @@ apply plugin: 'releaseNotes'
 
 
 group = 'com.spidasoftware'
-version = '6.0.0-SNAPSHOT'
+version = '6.0.15-SNAPSHOT' // If you're changing the major version also change the currentVersion in ConverterUtils.
```
Comment (spidadeveloper) [inline build.gradle]: Set this back.

File: doc/apis/spidadbAPI.md [hunk 4]
```diff
@@ -188,6 +188,23 @@ The response:
...
+## Detailed Results
+
+Starting with the version 6 schema detailed results will be pushed from SPIDAcalc to SPIDAdb.  The detailed results are very large json objects so SPIDAdb will not send the detailed results nested in the objects returned from the API.  Instead a uri will be returned, you will have to make an extra request to retrieve the detailed results.
+
+An example of a design with a uri:
+
+    {
+    	"analysis": [
+		{
+        		"uri": "588a58e17d84ad3bd41c4562"
```
Comment (spidadeveloper) [inline doc/apis/spidadbAPI.md]: id

File: doc/apis/spidadbAPI.md [hunk 5]
```diff
@@ -188,6 +188,23 @@ The response:
...
+		}
```
Comment (spidadeveloper) [inline doc/apis/spidadbAPI.md]: What about the calc API?

File: src/main/groovy/com/spidasoftware/schema/conversion/ExchangeFile.groovy [hunk 6]
```diff
@@ -106,6 +113,14 @@ public class ExchangeFile {
 				Files.copy(photo, target)
 			}
 		}
+		if(detailedResults) {
```
Comment (spidadeveloper) [inline src/main/groovy/com/spidasoftware/schema/conversion/ExchangeFile.groovy]: Get the stream for the id and copy that.

File: build.gradle (hunk 3 above)
Comment (spidadeveloper) [inline build.gradle]: Changeset to remove [resultId: id] from analysis list.

**Pull request merged**